src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Models/CmdletParamEnums.cs

```diff
@@ -50,13 +50,27 @@ public enum RecoveryPointParams
     public enum RestoreBackupItemParams
     {
         RecoveryPoint,
-        StorageAccountId,
-        StorageAccountLocation,
-        StorageAccountType,
+        StorageAccountName,
+        StorageAccountResourceGroupName
+    }
+
+    public enum RestoreVMBackupItemParams
+    {
         TargetResourceGroupName,
         OsaOption
     }
 
+    public enum RestoreFSBackupItemParams
+    {
+        ResolveConflict,
+        SourceFilePath,
+        SourceFileType,
+        TargetStorageAccountName,
+        TargetStorageAccountResourceGroupName,
```

Comment:
Based on discussion, please consider using Resources SDK to fetch the Storage Account RG